File: src/main/java/com/atsuishio/superbwarfare/config/client/ControlConfig.java
package com.atsuishio.superbwarfare.config.client;

import net.neoforged.neoforge.common.ModConfigSpec;

public class ControlConfig {

    public static ModConfigSpec.BooleanValue INVERT_AIRCRAFT_CONTROL;
    public static ModConfigSpec.IntValue MOUSE_SENSITIVITY;

    public static void init(ModConfigSpec.Builder builder) {
        builder.push("control");

        builder.comment("Set true to invert aircraft control");
        INVERT_AIRCRAFT_CONTROL = builder.define("invert_aircraft_control", false);

        builder.comment("Sensitivity of mouse");
        MOUSE_SENSITIVITY = builder.defineInRange("mouse_sensitivity", 100, 10, 200);

        builder.pop();
    }
}

File: src/main/java/com/atsuishio/superbwarfare/config/client/ControlConfig.kt
package com.atsuishio.superbwarfare.config.client

import com.atsuishio.superbwarfare.config.buildClientConfig

object ControlConfig {

    @JvmField
    val INVERT_AIRCRAFT_CONTROL = buildClientConfig {
        push("control")

        comment("Set true to invert aircraft control")
        define("invert_aircraft_control", false)
    }

    @JvmField
    val MOUSE_SENSITIVITY = buildClientConfig {
        comment("Sensitivity of mouse")
        defineInRange("mouse_sensitivity", 100, 10, 200).also { pop() }
    }
}
